### DISCUSSION TITLE
Curiosity: how does Spirit's character work?

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#503] [Q&A]
I noticed that every character image has an XML EXCEPT for spirit who randomly has a TXT file with completely different format. Why and how does this work?

**Comment by NeeEoo** [comment]
Ask FunkinCrew, this isnt our thing, we just support it for compatibility

**Reply by THEkatinamicrowave** [reply]
ok
